src/main/java/uk/gov/companieshouse/api/accounts/enumeration/AccountingNoteType.java

```diff
@@ -6,76 +6,96 @@
-                    NoteType.ACCOUNTING_POLICIES,
-                    SmallFullLinkType.ACCOUNTING_POLICY_NOTE,
-                    "small-full-accounts-note#accounting-policy",
-                    false),
+            AccountType.SMALL_FULL,
+            NoteType.ACCOUNTING_POLICIES,
+            Parent.SMALL_FULL,
+            SmallFullLinkType.ACCOUNTING_POLICY_NOTE,
+            "small-full-accounts-note#accounting-policy",
+            false),
     SMALL_FULL_EMPLOYEES(
-                    AccountType.SMALL_FULL,
-                    NoteType.EMPLOYEES,
-                    SmallFullLinkType.EMPLOYEES_NOTE,
-                    "small-full-accounts-note#employees",
-                    true),
+            AccountType.SMALL_FULL,
+            NoteType.EMPLOYEES,
+            Parent.SMALL_FULL,
+            SmallFullLinkType.EMPLOYEES_NOTE,
+            "small-full-accounts-note#employees",
+            true),
     SMALL_FULL_INTANGIBLE_ASSETS(
-                    AccountType.SMALL_FULL,
-                    NoteType.INTANGIBLE_ASSETS,
-                    SmallFullLinkType.INTANGIBLE_ASSETS_NOTE,
-                    "small-full-accounts-note#intangible-assets",
-                    true),
+            AccountType.SMALL_FULL,
+            NoteType.INTANGIBLE_ASSETS,
+            Parent.SMALL_FULL,
+            SmallFullLinkType.INTANGIBLE_ASSETS_NOTE,
+            "small-full-accounts-note#intangible-assets",
+            true),
     SMALL_FULL_TANGIBLE_ASSETS(
-                    AccountType.SMALL_FULL,
-                    NoteType.TANGIBLE_ASSETS,
-                    SmallFullLinkType.TANGIBLE_ASSETS_NOTE,
-                    "small-full-accounts-note#tangible-assets",
-                    true),
+            AccountType.SMALL_FULL,
+            NoteType.TANGIBLE_ASSETS,
+            Parent.SMALL_FULL,
+            SmallFullLinkType.TANGIBLE_ASSETS_NOTE,
+            "small-full-accounts-note#tangible-assets",
+            true),
     SMALL_FULL_FIXED_ASSETS_INVESTMENTS(
-                    AccountType.SMALL_FULL,
-                    NoteType.FIXED_ASSETS_INVESTMENTS,
-                    SmallFullLinkType.FIXED_ASSETS_INVESTMENTS_NOTE,
-                    "small-full-accounts-note#fixed-assets-investments",
-                    true),
+            AccountType.SMALL_FULL,
+            NoteType.FIXED_ASSETS_INVESTMENTS,
+            Parent.SMALL_FULL,
+            SmallFullLinkType.FIXED_ASSETS_INVESTMENTS_NOTE,
+            "small-full-accounts-note#fixed-assets-investments",
+            true),
     SMALL_FULL_STOCKS(
-                    AccountType.SMALL_FULL,
-                    NoteType.STOCKS,
-                    SmallFullLinkType.STOCKS_NOTE,
-                    "small-full-accounts#stocks-note",
-                    true),
+            AccountType.SMALL_FULL,
+            NoteType.STOCKS,
+            Parent.SMALL_FULL,
+            SmallFullLinkType.STOCKS_NOTE,
+
+            "small-full-accounts#stocks-note",
```

Comment:
Should this be 

> small-full-accounts#stocks

Reply:
Unfortunately not... I've had to maintain consistency with the `Kind`s we already used - this is stocks: https://github.com/companieshouse/company-accounts.api.ch.gov.uk/blob/master/src/main/java/uk/gov/companieshouse/api/accounts/Kind.java#L16